directorynameproject2/directorynameproject2/Questionaire.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Questionaire
+{
+    public class Questionaire //the properties are here. these are the blueprint of your methods. cookie cutter, not the cookie
```

Comment:
"Questionaire" seem like an odd name for this class. It's not really modeling a questionnaire so much as it's modeling a person or a user.  A user has a first name, last name, etc.  A questionnaire would ask for these things, but not really have them as properties of itself